connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1768,6 +1917,7 @@ ble_error_t Gap::setAdvertisingParameters(
 #endif // BLE_FEATURE_EXTENDED_ADVERTISING
     {
         if (handle != LEGACY_ADVERTISING_HANDLE) {
+            tr_error("Failed: non-existent advertising set");
```

Comment:
Bellow an error is not handled, you should add an else after the `if(!err)`

Reply:
I see. Thanks for pointing that out.